### PR TITLE
gcc: import upstream patch for bug 90756

### PR DESCRIPTION
**Pull request by jnbr**
This patch is necessary to build firefox 68 on i686.
Apparently only i686 and s390x are affected by this bug, so only patch the i686 cross tool chains.

### CODE REVIEW

**Comment by pullmoll** [comment]
LGTM

File: srcpkgs/cross-i686-linux-musl/files/bug90756.patch [hunk 1]
```diff
@@ -0,0 +1 @@
+../../gcc/patches/bug90756.patch
```
Comment (pullmoll) [inline srcpkgs/cross-i686-linux-musl/files/bug90756.patch]: This needs to be applied in the template. There is no auto patching in `cross-*`.

File: srcpkgs/cross-i686-pc-linux-gnu/files/bug90756.patch [hunk 2]
```diff
@@ -0,0 +1 @@
+../../gcc/patches/bug90756.patch
```
Comment (pullmoll) [inline srcpkgs/cross-i686-pc-linux-gnu/files/bug90756.patch]: Same here. Add `_apply_patch -p0 ${FILESDIR}/bug90756.patch` in `_gcc_bootstrap()`.

File: srcpkgs/cross-i686-linux-musl/files/bug90756.patch (hunk 1 above)
Comment (jnbr) [reply srcpkgs/cross-i686-linux-musl/files/bug90756.patch]: Oops, fixed.

**Pull request merged**